src/reducers/pause.js

```diff
@@ -299,7 +300,7 @@ function getPauseLocation(state, action) {
 // top-level app state, so we'd have to "wrap" them to automatically
 // pick off the piece of state we're interested in. It's impossible
 // (right now) to type those wrapped functions.
-type OuterState = { pause: PauseState };
+type OuterState = State;
```

Comment:
So... this could be really cool, let me explain:

Our selectors in the reducers use `OuterState` as a hack to pretend they receive the full AppState type. Now it looks like we can just import that type from `reducers/types`